.code-samples.meilisearch.yaml

```diff
@@ -1287,6 +1287,31 @@ analytics_event_bind_event_1: |-
       "objectId": "0",
       "position": 0
     }'
+search_parameter_reference_distinct_1: |-
+  curl \
+    -X POST 'http://localhost:7700/indexes/INDEX_NAME/search' \
+    -H 'Content-Type: application/json' \
+    --data-binary '{
+      "q": "QUERY TERMS",
+      "distinct": "ATTRIBUTE_A"
+    }'
+distinct_attribute_guide_filterable_1: |-
+  curl \
+    -X PUT 'http://localhost:7700/indexes/products/settings/filterable-attributes' \
+    -H 'Content-Type: application/json' \
+    --data-binary '[
+      "product_id",
+      "sku",
+      "url"
+    ]'
+distinct_attribute_guide_distinct_parameter_1: |-
+  curl \
+    -X POST 'http://localhost:7700/indexes/products/search' \
+    -H 'Content-Type: application/json' \
+    --data-binary '{
+      "q": "white shirt",
+      "distinct": "product_id"
```

Comment:
I think SKU is a more common example for people demanding distinct, but maybe I'm wrong, so you can check with product team

Reply:
Sure, I can change it. It's a purely illustrative example showing how one would use it in a "real life" setting, so it's more of a question of it being somewhat credible on a quick glance.

Reply:
This is where product team can help you with a real use case 👍